modules/delegates/components/DelegateAvatarName.tsx

```diff
@@ -0,0 +1,47 @@
+import { Delegate } from '../types';
```

Comment:
Refactored the delegate avatar + name into a component for reusing between comments and delegate cards